File: backend/components/metiers/administration_publique/utils/exporter/samples/sample_exporter_helper.test.js
// Test du helper d’export d’exemple (JS)
const { sampleExport } = require('./sample_exporter_helper');
describe('sampleExport', () => {
  it('should log and return true', () => {
    expect(sampleExport({id: 1, name: 'Test'})).toBe(true);
  });
});

File: backend/components/metiers/administration_publique/utils/exporter/samples/sample_exporter_helper.test.py
# Test du helper d’export d’exemple (Python)
from .sample_exporter_helper import sample_export
def test_sample_export(capsys):
    assert sample_export({'id': 1, 'name': 'Test'}) is True
    captured = capsys.readouterr()
    assert '[EXPORT] Données exportées:' in captured.out
